.github/workflows/github.yml

```diff
@@ -115,14 +115,12 @@ jobs:
 
     - name: Test Installation
       run: |
-        #$WebClient = New-Object System.Net.WebClient
-        #$WebClient.DownloadFile("https://cgal.geometryfactory.com/~mgimeno/cgal-5.0.2-cp37-cp37m-win_amd64.whl", "C:\cgal-5.0.2-cp37-cp37m-win_amd64.whl")
-        #pip install C:\cgal-5.0.2-cp37-cp37m-win_amd64.whl
         cd examples/python
-        Get-ChildItem "$PWD" |
-        Foreach-Object {
-          python.exe $_
-        }
+        for py_file in *.py; do
+          python.exe $py_file || exit 1
+        done
+      shell: bash
```

Comment:
Can this work without any call to `pip install ...`?

Reply:
pip install is called at the end of the previous job.